operator/config/crd/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+# This kustomization.yaml is not intended to be run by itself,
```

Comment:
Are these kubsomize things relevant to odigos? do we need to keep them?

Reply:
yeah, this is how the ready-to-deploy operator manifests are built and packaged into the bundle with operator-sdk/kubebuilder